Sources/XcbeautifyLib/Renderers/GitHubActionsRenderer.swift

```diff
@@ -130,6 +130,11 @@ struct GitHubActionsRenderer: OutputRendering {
         let reason = group.reason
         return outputGitHubActionsLog(annotationType: .error, message: reason)
     }
+    
+    func formatLinkerDuplicateSymbolsLocation(group: LinkerDuplicateSymbolsLocationCaptureGroup) -> String? {
+        let filePath = group.filePath
+        return outputGitHubActionsLog(annotationType: .warning, message: filePath)
```

Comment:
Looks like you're supplying the wrong arguments here. You'll want something something like...

```swift
let fileComponents = group.filePath.asFileComponents()
let message = group.message // or whatever other property holds the warning

return outputGitHubActionsLog(
    annotationType: .warning,
    fileComponents: fileComponents,
    message: message
)
```

Reply:
Got it.

Reply:
If that's the case, then I think we'd want to parse the warning as a message rather than a `filePath`.